web/models/user.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Shareguru.User do
+  use Shareguru.Web, :model
+
+  schema "users" do
+    field :name, :string
+    field :email, :string
+    field :picture, :string
+    field :expire_at, Ecto.DateTime
+
+    timestamps
+  end
+
+  @required_fields ~w(name email picture)
+  @optional_fields ~w(expire_at)
+
+  @doc """
+  Creates a changeset based on the `model` and `params`.
+
+  If no params are provided, an invalid changeset is returned
+  with no validation performed.
+  """
+  def changeset(model, params \\ :empty) do
+    model
+    |> cast(params, @required_fields, @optional_fields)
+    |> validate_format(:email, ~r/@/)
```

Comment:
We could validate for `@netguru` domain. Unless we don't really aim for 1:1 shareguru clone :-)